src/commands/commerce/extension/map.ts

```diff
@@ -0,0 +1,132 @@
+        );
+
+        if (results instanceof SfdxError) {
+            throw new SfdxError(msgs.getMessage('extension.map.error', [extensionName, '\n', results.message]));
+        }
+        // JSON response of inserted record
+        return this.getInsertedRecord(storeid, registeredExternalServiceId);
+    }
+
+    private getExtensionName(extensionName: string, userName: string): string {
+        let registeredExternalServiceId: string;
+        try {
+            registeredExternalServiceId = forceDataSoql(
+                `SELECT Id FROM RegisteredExternalService WHERE DeveloperName='${extensionName}'`,
+                userName
+            ).result.records[0].Id;
+        } catch (error) {
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+            const errorMsg = msgs.getMessage('extension.map.nonexistent', ['\n', error.message]);
+            throw new SfdxError(errorMsg);
+        }
+        return registeredExternalServiceId;
+    }
+
+    private getStoreId(storeName: string, storeId: string, userName: string): string {
+        if (storeId === undefined) {
+            try {
+                storeId = forceDataSoql(`SELECT Id FROM WebStore WHERE Name='${storeName}' LIMIT 1`, userName).result
+                    .records[0].Id;
+            } catch (e) {
+                throw new SfdxError(
+                    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+                    msgs.getMessage('extension.map.errStoreName', [storeName, '\n', e.message])
+                );
+            }
+        } else {
+            try {
+                storeId = forceDataSoql(`SELECT Id FROM WebStore WHERE Id='${storeId}' LIMIT 1`, userName).result
+                    .records[0].Id;
+            } catch (e) {
+                throw new SfdxError(
+                    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+                    msgs.getMessage('extension.map.errStoreId', [storeId, '\n', e.message])
+                );
+            }
+        }
+        return storeId;
+    }
+
+    private getInsertedRecord(storeid: string, registeredExternalServiceId: string): string {
+        const StoreIntegratedTable = forceDataSoql(
+            `SELECT Id,Integration,ServiceProviderType,StoreId from StoreIntegratedService WHERE StoreId= '${storeid}' and Integration='${registeredExternalServiceId}' limit 1`
+        );
+        for (const element of StoreIntegratedTable.result.records) {
+            const finalTable = {
+                Id: element['Id'],
+                Integration: element['Integration'] as string,
+                StoreId: element['StoreId'] as string,
+            };
+            const returnResult = `${JSON.stringify(finalTable, null, 4)}\n`;
+            this.ux.log(returnResult);
+            this.ux.log(
+                msgs.getMessage('extension.map.savingConfigIntoConfig', [this.flags['registered-extension-name']])
```

Comment:
currently, the success message makes it seem like the registered extension name is the webstore name
`Successfully mapped the extension to webstore: JinsuTicket`

could we change it to 
`Successfully mapped the extension 'JinsuTicket' to the webstore '1commerce'`